### ISSUE TITLE
Use "Rome Frontend" in favor of Babel, ESLint, webpack, Prettier, Jest and others

### ISSUE THREAD

**Issue by Vadorequest** [UnlyEd/next-right-now#152]
I just discovered https://github.com/romefrontend/rome and it looks very promising.

I have no idea how it can work with Next.js.

I'm concerned about how it could be breaking with webpack. But interested about how it could replace eslint and prettier (we don't use prettier, mostly because of https://github.com/prettier/prettier/pull/6644). I like the idea of having one tool to deal with all those boring things, as long as it does the job well.

Also, it would require to convert the whole project from "2 spaces" to tabs. But this is better for developer accessibility anyway.

The plan is to see how it can work with Next and see if it can be integrated within NRN then.

Rome is built by Facebook and Babel, according to https://www.infoq.com/news/2020/03/rome-experimental-js-toolchain/

Discussion opened on Next.js community: https://github.com/vercel/next.js/discussions/16754

**Comment by samuelcastro** [comment]
I really like the idea and it does look promising, however I don't think it's ready yet.

I'm curious about how next.js would work with it though.

**Comment by Vadorequest** [comment]
Yeah, it's too early clearly. It can only be used as a linter for now and needs to mature quite a bit to be able to replace all those tools. But the philosophy/vision is really great. I'll keep an eye on it.

**Comment by Vadorequest** [comment]
FYI Rome started a fundraising today, goal is to reach $100k. 

https://rome.tools/funding/

**Comment by Vadorequest** [comment]
I've discovered https://github.com/brodybits/prettierx which is less opinionated than prettier.

**Comment by samuelcastro** [comment]
I think that if you want to have consistent code acros the board with standardized rules, it does make more sense to use a more opinionated tool. This is why I like Prettier, I'd rather use it.

**Comment by Vadorequest** [comment]
Rome has raised $4.5 million (seed).

https://rome.tools/blog/announcing-rome-tools-inc/

**Comment by Vadorequest** [comment]
https://rome.tools/blog/2022/11/08/rome-10.html first stable release
